LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -187,6 +187,305 @@ function groupListContent(groups) {
     return content;
 }
 
+// Automated Emails Section
+function emptyAlert(idElement) {
+    const elementId = document.getElementById(idElement).value;
+    if (elementId == "") {
+        alert("Please make a choice");
+    }
+}
+
+function createElement(typeOfElement, elementId, parentDiv) {
```

Comment:
What kind of error would be generated if typeOfElement is null?

Reply:
Going to ask you to look over all these functions and make sure not having a default value isn't going to cause unintended consequences.